src/main/resources/default/assets/common/core.js.pasta

```diff
@@ -30,6 +30,22 @@ sirius.isEmptyNode = function (node) {
     return false;
 }
 
+/**@
+ * Checks if a given DOM node is visible.
```

Comment:
Hmm, I would like to have the documentation extended a bit, to describe what visibility is in this context. Apparently, it does not mean "visible in the viewport“ here, i.e. the method checks only for whether a node _could_ be visible, not whether it is currently shown to the user.

Reply:
Better?